### ISSUE TITLE
Row break not working and different in 3.1.1 and 3.2.2

### ISSUE THREAD

**Issue by t-book** [simonbengtsson/jsPDF-AutoTable#532]
Hi,

I'm fighting with the rendering of content-rich tables. One cell has text which should overflow to the next page. Unsure if this has been already reported but it looks like a bug to me.

**jsPDF-AutoTable3.1.1**
https://codepen.io/t-book/pen/OKBQbG

Will correctly cut the row and put in on the next page. Unfortunately
 the bottom border calculation is wrong and text runs out of the table.

 ![Bildschirmfoto 2019-08-11 um 11 28 58](https://user-images.githubusercontent.com/20478652/62832233-83102600-bc2b-11e9-868a-92d7d6eda646.png)

**jsPDF-AutoTable-3.2.2**
https://codepen.io/t-book/pen/voVdrb
Does not break the row anymore. The Text is just cutoff. 

While I could live with the behavior of 3.1.1 I would be curious if it's possible to avoid the text-overflow or if I'm missing something with version 3.2.2? The pens will show the used configuration. II will put the interesting part here:

```
    doc.autoTable({
        columnStyles: { 0: {halign: 'left', minCellWidth: 50, fontStyle: 'bold', textColor: [0, 0, 0], font: 'csv-b'},
                        1:{halign: 'left',textColor: [0, 0, 0], minCellWidth: 135,font: 'csv-m'}}, 
        margin: {top: 30},
        pageBreak: 'avoid',
        rowPageBreak: 'avoid',
        theme: 'grid', 
```

Thanks a lot for your help and this great plugin,

Toni

**Comment by simonbengtsson** [comment]
I have seen multiple issues in this area lately so I'm working on rewriting the entire calculation for when to break pages etc. Especially regarding rowspans. Thanks for reporting and I'll take this example into consideration for the next release to make sure it is working 👍

**Comment by t-book** [comment]
Wonderful, thanks a lot @simonbengtsson

**Comment by simonbengtsson** [comment]
Fix is available in v3.2.3
